ms-docs/JBoss EAP on Azure.MD

```diff
@@ -133,35 +157,37 @@ You can deploy the template in three following ways:
 
-You can choose to start with one of the quickstart templates with required JBoss EAP on RHEL combination which meets your desired deployment goal. Following is the list of available quickstart templates.
+You can start with one of the Quickstart templates of JBoss EAP on RHEL combination that meets your deployment goal. Following is the list of available Quickstart templates.
 
-* <a href="https://github.com/Azure/azure-quickstart-templates/tree/master/jboss-eap-standalone-rhel" target="_blank"> JBoss EAP on RHEL (stand-alone VM)</a> - This Azure template deploys a web application named JBoss-EAP on Azure on JBoss EAP 7.2/EAP 7.3 running on RHEL 7.7/8.0 VM.
+* <a href="https://azure.microsoft.com/resources/templates/jboss-eap-standalone-rhel/"> JBoss EAP on RHEL (stand-alone VM)</a> - This Azure template deploys a web application named JBoss-EAP on Azure to JBoss EAP 7.2 or 7.3 running on RHEL 7.7 or 8.0 VM.
 
-* <a href="https://github.com/Azure/azure-quickstart-templates/tree/master/jboss-eap-clustered-multivm-rhel" target="_blank"> JBoss EAP on RHEL (clustered, multi-VM)</a> - This Azure template deploys a web application called eap-session-replication on JBoss EAP 7.2/EAP 7.3 cluster running on 'n' number RHEL 7.7/8.0 VMs where n is decided by the user and all the VMs are added to the backend pool of a Load Balancer.
+* <a href="https://azure.microsoft.com/resources/templates/jboss-eap-clustered-multivm-rhel/"> JBoss EAP on RHEL (clustered, multi-VMs)</a> - This Azure template deploys a web application called eap-session-replication on JBoss EAP 7.2 or 7.3 cluster running on 'n' number RHEL 7.7 or 8.0 VMs. The 'n' value is decided by the user. All the VMs are added to the backend pool of a Load Balancer.
 
-* <a href="https://github.com/Azure/azure-quickstart-templates/tree/master/jboss-eap-clustered-vmss-rhel" target="_blank"> JBoss EAP on RHEL (clustered, VMSS)</a> - This Azure template deploys a web application called eap-session-replication on JBoss EAP 7.2/EAP 7.3 cluster running on RHEL 7.7/8.0 VMSS instances.
+* <a href="https://azure.microsoft.com/en-us/resources/templates/jboss-eap-clustered-vmss-rhel/"> JBoss EAP on RHEL (clustered, Virtual Machine Scale Set)</a> - This Azure template deploys a web application called eap-session-replication on JBoss EAP 7.2 or 7.3 cluster running on RHEL 7.7 or 8.0 Virtual Machine Scale Set instances.
 
 ## Resource Links:
 
-* Explore [JBoss EAP on Azure Red Hat OpenShift](https://azure.microsoft.com/services/openshift/)
-* Explore [JBoss EAP on App Service Linux](https://docs.microsoft.com/azure/app-service/quickstart-java)
-* [Azure for Java developers documentation](https://github.com/JasonFreeberg/jboss-on-app-service)
+* [Azure Hybrid Benefits)(https://docs.microsoft.com/azure/virtual-machines/windows/hybrid-use-benefit-licensing)
+* [Configuring a Java app for Azure App Service](https://docs.microsoft.com/azure/app-service/configure-language-java?pivots=platform-linux)

```

Comment:
Changed this ink to "https://docs.microsoft.com/azure/app-service/configure-language-java"